ext/cfx-ui/src/assets/languages/locale-de.json

```
@@ -44,22 +51,35 @@
     "#Settings_MenuAudioDesc": "Aktiviere Hintergrundmusik während Inaktivität im Hauptmenü",
     "#Settings_GameStreamProgress": "Streaming-Fortschritt",
     "#Settings_GameStreamProgressDesc": "Zeigt den Fortschritt beim Herunterladen von im Spiel gestreamten Inhalten an. (BETA™)",
-    "#DirectConnect_TimedOut": "Zeitüberschreitung der Serveranfrage.",
+    "#Settings_QueriesPerMinute": "Serversuche: Maximale Pings / Minute",
+    "#Settings_CustomEmoji": "Eigenes FiveM Wasserzeichen",
+    "#Settings_CustomEmojiDesc": "Ein Emoji für das Wasserzeichen im Spiel.",
+    "#Settings_CustomEmojiUpsell": "Nicht vefügbar! Spende $5/Monat oder mehr, um einen offiziellen FiveM-Patreon Account zu bekommen und Dir Dein eigenes Emoji als Wasserzeichen zu setzen.",
+    "#Settings_ConnectedProfiles": "Verbundene Identitäten",
+    "#SettingsCat_Connection": "Verbindung",
+    "#SettingsCat_Interface": "Benutzeroberfläche",
+    "#SettingsCat_Game": "Spiel",
+    "#SettingsCat_Account": "Benuterkonto",
+    "#DirectConnect_TimedOut": "Zeitüberschreitung bei Serverabfrage.",
     "#DirectConnect_Failed": "Server konnte nicht abgefragt werden.",
-    "#DirectConnect_ServerInvalid": "Server konnte nicht gefunden werden",
-    "#DirectConnect_ServerInvalid_SubTitle": "Bitte überprüfe, ob du eine korrekte Adresse eingegeben hast.",
+    "#DirectConnect_Loading": "Lädt...",
+    "#DirectConnect_ServerInvalid": "Server kann nicht gefunden werden",
+    "#DirectConnect_ServerInvalid_SubTitle": "Stellen Sie sicher, dass Sie die richtige Adresse eingegeben haben.",
```

The change from informal (Du)  to formal (Sie) shouldn't be there. Keep it either informal or formal throughout.

```suggestion
    "#DirectConnect_ServerInvalid_SubTitle": "Stelle sicher, dass du eine gültige Adresse eingegeben hast.",
```